README.md

```diff
@@ -1,70 +1,26 @@
-
-### `yarn eject`
-
-**Note: this is a one-way operation. Once you `eject`, you can’t go back!**
-
-If you aren’t satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
-
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you’re on your own.
-
-You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
-
-## Learn More
-
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
-
-To learn React, check out the [React documentation](https://reactjs.org/).
-
-### Code Splitting
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/code-splitting](https://facebook.github.io/create-react-app/docs/code-splitting)
-
-### Analyzing the Bundle Size
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size](https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size)
-
-### Making a Progressive Web App
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app](https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app)
-
-### Advanced Configuration
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
-
-### Deployment
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
-
-### `yarn build` fails to minify
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+# AntRobtics Landing Page Overhaul
+
+## Dev Team
+Kevin Loughead
+Tanya Komar
+Jessica Perelman
+
+## Built With
+[Create React App](https://github.com/facebook/create-react-app)
+
+## Features
+• Sticky nav that links to sections on the page
+• Animated header
+• Carousel of robot options and holistic gallery
+• Cards detailing benefits and listing the team
+• Contact form for customer inquiries
+• Google Map API for location
+• Responsive based on multiple breakpoints
+See our [Trello Board](https://trello.com/b/CPA5Lprq/antrobotics) for the full breakdown!
+
```

Comment:
We'll need a description of the configurability too. I'll write something up for that. Maybe we could add this bullet point to this readme:

- Designed with configurability in mind. See the [configurability readme](config-readme.md) for details.

And then have the config details in a separate file? I think that link will work, but I'm not certain.

Reply:
Oo great idea, just added!

Reply:
Oo great idea, just added!